aten/src/ATen/native/native_functions.yaml

```diff
@@ -8904,6 +8904,20 @@
   dispatch:
     DefaultBackend: det
 
+- func: linalg_lstsq(Tensor self, Tensor b, float? cond=None, str? driver_name=None) -> (Tensor solution, Tensor rank, Tensor singular_values)
+  python_module: linalg
+  use_c10_dispatcher: full
+  variants: function
+  dispatch:
+    DefaultBackend: linalg_lstsq
+
+- func: _lstsq_helper(Tensor a, Tensor b, float cond, str? driver_name) -> (Tensor, Tensor, Tensor)
+  use_c10_dispatcher: full
+  variants: function
+  dispatch:
+    CPU: _lstsq_helper_cpu
+    CUDA: _lstsq_helper_cuda
```

Comment:
Maybe it makes sense to remove this from native functions and use the declared/defined dispatch instead? CC @mruberry

Reply:
Yep, that's a great idea.